### PR TITLE
fix: image list

### PR DESCRIPTION
**Pull request by alexcodelf**
**Problem:**
Missing airgap deployer image in list

**Solution:**

- Add airgap image to list.
- Release full image list in asset.

**Related Issue:**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -198,6 +198,10 @@ jobs:
           for file in cli*; do mv "$file" "$(echo "$file" | sed 's/^cli/walrus-cli/')"; done
           mkdir cli
           cp walrus-cli* cli/
+      - name: Copy Image list
+        if: ${{ github.event_name != 'pull_request' }}
+        run: |
+          cp hack/mirror/full_image_list.txt .dist/package/${{ matrix.target }}/${{ matrix.task }}/build/full_image_list.txt
```
Comment (gitlawr) [inline .github/workflows/ci.yml:204]: Rename previous step to `Prepare Release Assets` and merge these two steps

**Comment by thxCode** [comment]
1. Do not need to change anything in the original `Rename CLI Assets` step.
2. Rename the `Release CLI To Repository Asset` step. to `Release Asset`
3. Append one line to the `Release Asset` step: `hack/mirror/full_image_list.txt`.

**Comment by alexcodelf** [comment]
> 3\. hack/mirror/full_image_list.txt

updated

**Review by thxCode — APPROVED** [review]
LGTM

**Review by gitlawr — APPROVED** [review]

**Pull request merged**